### PR TITLE
added unittest for endpoint /languages POST

### PR DESCRIPTION
**Pull request by mkenlo**
To complete unittest for Languages I, added in this commit tests for POST /languages. I renamed the file from `test_endpoints` to `test_languages_endpoints` to have a single file test for each endpoint group

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -24,23 +24,24 @@ def index(request):
 
 @api.route("/languages", methods=["POST"])
 def saveLanguages(request):
-    if request.method == "POST":
-        postdata = request.json
-        requiredFields = ["language", "code", "type", "default"]
-        if not set(requiredFields) >= set(postdata):
-            return response.json(responseError, status=400)
```
Comment (evindj) [inline app/main.py:31]: remove this global object and instead create local variables everywhere.

File: tests/test_languages_endpoints.py [hunk 2]
```diff
@@ -83,3 +83,37 @@ def fin():
...
+
+async def test_addLanguages_with_validParam_shouldPass(test_cli):
+    params = {"language": "dummy_test", "code": "DUM", "type": "local"}
+    resp = await test_cli.post('/languages', data=json.dumps(params))
+    resp_json = await resp.json()
+    assert resp.status == 201
+    assert resp_json["message"] == "Added One Item"
+    assert Languages.objects().first().language == "dummy_test"
+    assert Languages.objects().first().code == "DUM"
+    removeCollections()
```
Comment (evindj) [inline tests/test_languages_endpoints.py:119]: I thought removeCollections() was called automatically when the test is ended?

**Review by evindj — APPROVED** [review]
please remove the offending global object before merging.

**Comment by mkenlo** [comment]
I updated as required

**Pull request merged**